src/onepay-direct-qr.js

```diff
@@ -1,9 +1,16 @@
 const OnepayWebSocket = require('./onepay-websocket.js');
 
-class Onepay {
+class OnepayDirectQr {
   constructor(transaction) {
-    this.version = '1.1.0';
-    this.transaction = transaction;
+    this.version = '1.2.0';
+
+    if (transaction) {
+      this.transaction = transaction;
```

Comment:
I don't understand why it's needed to do this, if transaction is not given then transaction will === undefined/null, which is basically the same result as not doing that ```if (transaction)```block

Reply:
What I mean is, if someone calls ```let directQr = new OnepayDirectQr();``` (no parameter given) then ```directQr.transactiom``` will be `undefined`, which is the same that would happen if you didn't do that ```if``` (unless someone called ```new OnepayDirectQr(false|null)```

Reply:
I think you are right so i will remove the if